addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -17,6 +17,13 @@ class ChatbotScriptStep(models.Model):
     _rec_name = 'message'
 
     message = fields.Text(string='Message', translate=True)
+    html_message = fields.Html(
```

Comment:
We want to avoid storing duplicate data if possible.
At least making the field non-stored would be sufficient, but the best alternative would be to send `message` to client without transforming it.

Reply:
If I'm not mistaken we don't need `plaintext2html`. Plain text sent to JS without markup will just be rendered as plain text anyway.

I guess we'd want to contain rich content, but then we could tweak the `_to_store` to enforce non-markup-ing the field. What do you think?

Reply:
But ideally we should just be sending the raw data to the JS and handle the transforming there.
Or even directly allow sanitized html content in step.